app/models/conference.rb

```diff
@@ -15,4 +17,15 @@ def tickets_left
     confirmed_attendances = attendances.select { |attendance| attendance.confirmed }
     tickets.to_i - confirmed_attendances.size
   end
+
+  private
+
+    def date_interval
+      if [starts_on, ends_on].count(&:nil?) == 1
```

Comment:
@klappradla We need a decision here about the validation of the dates for a conference. 
The main object for this PR is to not allow a `end date` input to be before the `start date`. 
If you can add your vote for either 'presence validation' or not, I can make the PR ready for review.